apps/sms/js/thread_ui.js

```diff
@@ -2490,7 +2490,9 @@ var ThreadUI = {
         message: message,
         onerror: function onError(error) {
           var errorName = error.name;
-          this.showMessageSendingError(errorName);
+          // Only sms will have FDN error
+          var recipient = error.data.type === 'sms' && [error.data.receiver];
+          this.showMessageSendingError(errorName, {recipients: recipient});
```

Comment:
Maybe it's just me, but it seems that if we skip message type checking it wouldn't hurt, but will make code simpler :
```
this.showMessageSendingError(
  error.name,
  { recipients: error.data.receivers || [error.data.receiver] }
);
```
I don't block on this, but what do you think?